src/libraries/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/PerformanceCounterLib.cs

```diff
@@ -79,6 +79,18 @@ private static object InternalSyncObject
             }
         }
 
+        private static CultureInfo EnglishCulture
+        {
+            get
+            {
+                if (s_englishCulture is null)
+                {
+                    try { s_englishCulture = CultureInfo.GetCultureInfo("en"); } catch { s_englishCulture = CultureInfo.InvariantCulture; }
```

Comment:
What could cause `GetCultureInfo("en")` to throw?

Reply:
If running with Globalization Invariant Mode, that will throw.

Reply:
Wouldn't it be better to check for invariant mode than to use exception for control flow? I remember you advised that we do that using `CultureInfo.GetCultureInfo("en-US").EnglishName.Contains("Invariant")` in SqlClient.

Reply:
`CultureInfo.GetCultureInfo("en-US")` can throw :-(